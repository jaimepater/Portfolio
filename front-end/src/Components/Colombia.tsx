import styled from 'styled-components';
import imageColombia from '../assets/Images/Colombia.svg';

const Colombia = styled.div`
  background-image: url(${imageColombia});
  height: 100%;
  width: 100%;
  background-size: 100% 100%;
`;

export default Colombia;
