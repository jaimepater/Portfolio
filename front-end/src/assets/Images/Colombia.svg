<?xml version="1.0" encoding="utf-8"?>
<svg width="65" height="49" viewBox="8.265 5.412 48.272 34.733" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
  <defs>
    <pattern id="pattern" preserveAspectRatio="none" width="1" height="1" viewBox="0 0 500 333">
      <image width="500" height="333" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAfQAAAFNCAYAAAD2E503AAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAakSURBVHhe7dxNilxVAIbhe6pBUNCOBlr8wcwUhYiTmI4zBQe6DWe6BCeZOHIg2P7hOlxHY+tEFLIHhyLW9XToBWSY8/I8dNV3bm3g5dZPj/+uzvYNAFja4WYBgIUJOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAEjOPV2X5zBgAWNfbLlwUdABY39qtXZtCP83j97ru11lprV9zxz+9n+zbmC/t8wVprrbVL7vj3z9v79XvuYz6stdZau+juj57b93k15pW11lprF92/H300jwDAysaXX30u6ACwuHF69zNBB4DFjWfvfi3oALC4Md77eQb9OI+H+bDWWmvtiju2+98JurXWWrv4ju2DbwXdWmutXXwPj3+8dn1hrbXW2mV3bA8ufCkOABY3Dve/F3QAWNzhcJNza6211q678w79wv9yt9Zaa1ffce4zdABY3eFmAYCFCToABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABAg6AAQIOgAECDoABIztwcV+cwYAFjXGuaADwOrGyfkPgg4AizsZr336cB/HbdvH/LPWWmvtinuyv/HJw/m8Pb5Nt9Zaa+2S60txABAwnnn/J0EHgMWN5+/9OIN+nMfrn6Rba621dsUdL937Zt/GbPo+Zt6ttdZau+KOD9/8Yl4BACsbv9z5WNABYHHjrxfu7Md5t36YWbfWWmvtmjv+uP36/oRvz1trrbX2ad2r01f3J/8OnbXWWmufxh2/3XrHHbq11lq7+v56+u48AgArG5e33hZ0AFjcuHzxLUEHgMVdf5YOACxO0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABIEDQASBA0AEgQNABYHnb9j/QkqAnPtZtkAAAAABJRU5ErkJggg=="/>
    </pattern>
    <filter id="Imagen_1" x="0" y="0" width="65" height="49" filterUnits="userSpaceOnUse">
      <feOffset dy="3" input="SourceAlpha"/>
      <feGaussianBlur stdDeviation="3" result="blur"/>
      <feFlood flood-opacity="0.161"/>
      <feComposite operator="in" in2="blur"/>
      <feComposite in="SourceGraphic"/>
    </filter>
  </defs>
  <g transform="matrix(1, 0, 0, 1, 0, 0)" filter="url(#Imagen_1)">
    <rect id="Imagen_1-2" data-name="Imagen 1" width="47" height="31" rx="6" transform="translate(9 6)" fill="url(#pattern)"/>
  </g>
</svg>