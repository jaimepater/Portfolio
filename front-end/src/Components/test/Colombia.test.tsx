import React from "react";
import { shallow } from 'enzyme';
import Colombia from "../Colombia";


describe('Colombia', () => {
  it('Renders Colombia', () => {
    const shallowComponent = shallow(<Colombia />);
    expect(shallowComponent).toMatchSnapshot();
  });
});