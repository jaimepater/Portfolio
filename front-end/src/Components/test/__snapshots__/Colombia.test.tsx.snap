// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Colombia Renders Colombia 1`] = `
<div
  className="sc-AykKC fclvYn"
/>
`;
